tests/test-collection/antsibull-nox.toml

```
@@ -6,3 +6,20 @@
 
 [sessions.lint]
 run_mypy = false
+
+[[sessions.ee_check.execution_environments]]
```

If the EE sessions should actually run by default, you should mark them with `default = true`:
```suggestion
[sessions.ee_check]
default = true

[[sessions.ee_check.execution_environments]]
```
(This is also not documented in docs/config-file.md right now.)